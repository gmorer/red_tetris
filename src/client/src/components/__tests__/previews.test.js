import Previews from './../previews';
import React from 'react';
import renderer from 'react-test-renderer';

test('Some test', () => {
	const component = renderer.create(<Previews previews={[]} />)
	const tree = component.toJSON();
	expect(tree).toMatchSnapshot();
})