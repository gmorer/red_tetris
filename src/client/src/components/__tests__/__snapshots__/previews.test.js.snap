// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some test 1`] = `
<div
  style={
    Object {
      "alignSelf": "center",
      "display": "flex",
      "flexFlow": "row wrap",
      "justifyContent": "space-around",
      "marginRight": "3em",
      "marginTop": "5em",
    }
  }
>
  <div
    style={
      Object {
        "color": "white",
        "marginBot": "2em",
        "marginLeft": "3em",
        "marginRigth": "3em",
        "marginTop": "2em",
        "position": "relative",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundColor": "white",
          "height": "24em",
          "width": "12em",
        }
      }
    >
      <div
        style={
          Object {
            "display": "flex",
            "flexDirection": "column",
            "height": "100%",
            "marginTop": "-40%",
            "width": "100%",
          }
        }
      >
        <div
          style={
            Object {
              "display": "flex",
              "flexDirection": "row",
              "height": "5%",
              "width": "100%",
            }
          }
        >
          <div
            style={
              Object {
                "backgroundColor": "transparent",
                "height": "100%",
                "width": "10%",
              }
            }
          />
          <div
            style={
              Object {
                "backgroundColor": "transparent",
                "height": "100%",
                "width": "10%",
              }
            }
          />
        </div>
      </div>
    </div>
    toto
  </div>
</div>
`;
